### PR TITLE
Upgrade version

### LINKED ISSUE #31: not compatible with new IntelliJ IDEA 2023.1
This message appears since last IDEA update:
Plugin 'Halcyon Theme' (version '2.3.1') is not compatible with the current version of the IDE, because it requires build 223.* or older but the current build is IC-231.8109.175

### PR DESCRIPTION
**Pull request by springmonster**
Hi @ADMARIl , I'm trying to resolve #31 .
Could you please to help review this PR?
Thanks!

### CODE REVIEW

**Comment by BrianVia** [comment]
Hey @ADMARIl I feel like I've noticed this isn't a top priority for you (totally understand, we all have lots of things on our plate), I'd love to volunteer as a maintainer if it'd be possible to merge PRs and push new releases of this theme 🙋‍♂️

I believe I speak for the community when I say we really love it and want to help take some load of the work if it'd help get releases for new versions of Jetbrains IDEs pushed out as reasonably close behind the new versions of IDEs.

Thank you again for all the work you've done maintaining this ❤️

**Comment by ADMARIl** [comment]
@BrianVia Sure, works for me

File: libs/halcyon-jetbrains-2.3.2.jar
Comment (ADMARIl) [inline libs/halcyon-jetbrains-2.3.2.jar:1]: I understand your reasoning for adding this, however, I feel that we should avoid having derivative files checked in. Release jars are best saved for adding to releases imo

**Review by ADMARIl — CHANGES_REQUESTED** [review]

File: libs/halcyon-jetbrains-2.3.2.jar
Comment (springmonster) [reply libs/halcyon-jetbrains-2.3.2.jar:1]: Yes，you are right. That's not a good solution. I will delete that comment.

Comment (springmonster) [reply libs/halcyon-jetbrains-2.3.2.jar:1]: Hi @ADMARIl , please help to review.

**Comment by BrianVia** [comment]
Hey @ADMARIl , are you able to share the publish token with me? My email is brian.via.dev@gmail.com if you want to send a secure link there.

**Comment by ADMARIl** [comment]
@BrianVia done. Let me know if there's issues publishing the release

**Pull request merged**